style_rwd.css

```diff
@@ -1,22 +1,67 @@
 @media (max-width: 767px){
     img.main_photo {
         display: none;
+        width:auto;
     }
-    .catch_phrase{
-        max-width: 767px;
+    .section_hero {
+        padding-bottom: 42px;
+    }
+    body{
+        width: auto;
     }
     .column{
+        width: auto;
         align-items: center;
         justify-content: center;
     }
-    .invite_phrase{
+    .catch_phrase .invite_phrase{
         text-align: center;
-        font-size: 5px;
+        font-size: 40px;
         font-weight: 30;
-        max-width: 767px;
+        margin-bottom: 0;
+        width: auto;
+    }
+    .button_container {
+        justify-content: center;
+        width: auto;
+        margin: 0;
     }
-    .btn{
+    .btn {
+        border-radius: 32px;
+        padding: 5px 16px;
         font-size: 12px;
     }
-}
+    .section_container {
+        min-height: 400px;
+        justify-content: center;
+    }
+    .catch_phrase {
+        flex-direction: column;
+        width: auto;
+        line-height: 95px;
+    }
+
+    .flex_column {
+        flex-direction: column;
+    }
+
+    .section_heading {
+        margin: auto;
```

Comment:
what is margin: auto?

Reply:
it sets margin for auto acc to device width?

Reply:
set for 40px

Reply:
☝️